lib/node_modules/@stdlib/math/base/special/maxabsf/test/test.js

```diff
@@ -0,0 +1,94 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnanf = require( '@stdlib/math/base/assert/is-nanf' );
+var isPositiveZerof = require( '@stdlib/math/base/assert/is-positive-zerof' );
+var float64ToFloat32 = require( '@stdlib/number/float64/base/to-float32' );
+var PINF = require( '@stdlib/constants/float32/pinf' );
+var maxabsf = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof maxabsf, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `NaN` if provided a `NaN`', function test( t ) {
+	var v;
+
+	v = maxabsf( NaN, 3.14 );
+	t.strictEqual( isnanf( v ), true, 'returns expected value' );
+
+	v = maxabsf( 3.14, NaN );
+	t.strictEqual( isnanf( v ), true, 'returns expected value' );
+
+	v = maxabsf( NaN );
```

Comment:
We have this in [`maxabs`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/maxabs/test/test.js#L47) as well.  In `test.native.js`, it is not present, as it gives errors because we provide only one argument instead of two. But no error comes if it is present in `test.js`. Should we remove this test here and in `maxabs` as well?

Reply:
I am confused about why no error is raised for the `js` test here.

Reply:
I believe that test is simply missing a second `NaN`. This would allow for testing all `NaN` combinations.

Reply:
In which case, the test should be added to `test.native.js` and fixed in the other package(s).

Reply:
Thanks!